react-file-uploads/client/app.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+
+    const formData = new FormData(event.target);
+
+    console.log(formData)
+
+    const req = {
+      method: 'POST',
+      body: formData
+    };
+    fetch('/api/uploads', req)
+      .then(res => {
+        console.log(res)
+      })
+      .catch(err => next(err));
+    event.target.reset()
```

Comment:
You don't want to reset the form until you have a successful response from the server, and you need to parse your response:

```JAVASCRIPT
fetch('/api/uploads', req)
  .then(res => res.json())
  .then(data => {
    console.log(data)
    event.target.reset()
  })
  .catch(err => next(err));
```